ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -236,6 +236,41 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                                     options:(ORKPredefinedTaskOption)options;
 
 /**
+ Returns a predefined task that enables an audio recording WITH a check of the audio level.
+ 
+ In an audio recording task, the participant is asked to make some kind of sound
+ with their voice, and the audio data is collected.
+ 
+ An audio task can be used to measure properties of the user's voice, such as
+ frequency range, or the ability to pronounce certain sounds.
+ 
+ Data collected in this task consists of audio information.
+ 
+ @param identifier              The task identifier to use for this task, appropriate to the study.
+ @param intendedUseDescription  A localized string describing the intended use of the data
+ collected. If the value of this parameter is `nil`, default
+ localized text is used.
+ @param speechInstruction       Instructional content describing what the user needs to do when
+ recording begins. If the value of this parameter is `nil`,
+ default localized text is used.
+ @param shortSpeechInstruction  Instructional content shown during audio recording. If the value of
+ this parameter is `nil`, default localized text is used.
+ @param duration                The length of the count down timer that runs while audio data is
+ collected.
+ @param recordingSettings       See "AV Foundation Audio Settings Constants" for possible values.
+ @param options                 Options that affect the features of the predefined task.
+ 
+ @return An active audio task that can be presented with an `ORKTaskViewController` object.
+ */
++ (ORKOrderedTask *)audioLevelNavigableTaskWithIdentifier:(NSString *)identifier
```

Comment:
Existing apps already use this task with the assumption that it is an `ORKOrderedTask` and *not* an `ORKNavigableOrderedTask`. Because the task steps in an `ORKOrderedTask` are immutable (and the publicly exposed accessor is *not* used internally), any app that needs to mutate the task has to do so by creating this task and then mutating the steps and instantiating a *new* `ORKOrderedTask`. Adding the branching step will break those implementations. That said, I realize that the method signature should explicitly return an `ORKNavigableOrderedTask` and will update the method signature.